README.md

```diff
@@ -81,15 +93,58 @@ Please add below permissions keys to your `Info.plist` file with proper descript
 
 Please enable `Background Modes` in your project `Signing & Capibilities` tab. After enabling please check box with option `Audio, Airplay, and Pictures in Pictures`. If you don't enables this setting, your mic will be muted when your app goes to background.
 
+### Enable Audio Video Permissons
+
+Before joining the meeting please check audio video permissons are enabled or not. If not please throw an error to enable both audio and video permissons 
 
 ## Integration Steps
 
 ### Add SDK
 
-Please add below pod to your Podfile and run command `pod install --repo-update`.
+Please add below pod to your Podfile and run command `pod install`.
+
+```ruby
+pod 'JioMeetUIKit_iOS', '2.5'
+```
+
+To enable paritipant panel we need to set `JMUIKit.isParticipantPanelEnabled = true` and add below pod to your Podfile
```

Comment:
Change optional frameworks description like below

JioMeetUIKit has many optional features which you can enable according to your requirement. Please check below snippet.

 

```swift

// Enable White Board Feature. You can ignore it if you don't want to use Whiteboard

JMUIKit.isWhiteboardEnabled = true

 

// Enable Start and Stop Recording option. You can ignore if you don't want to handle recording from app side

JMUIKit.isRecordingEnabled = true

 

// Enable Entry/Exit Chime. You can ignore if you don't want to use this feature

// Please provide path for Entry/Exit Chime audio files if you enable this feature

JMUIKit.isEntryExitChimeEnabled = true

JMUIKit.entryChimeSoundPath = Bundle.main.path(forResource: "user_join_chime", ofType: "mp3") ?? ""

JMUIKit.exitChimeSoundPath = Bundle.main.path(forResource: "user_left_chime", ofType: "mp3") ?? ""

```